web/src/beta/lib/core/Map/featureFlags.ts

```diff
@@ -0,0 +1,7 @@
+export const MAP_FEATURE_FLAGS = {
```

Comment:
Two questions:
- Isn't this an object? Why are you calling it a map?
- If it is somehow a map, I think calling it FEATURE_FLAGS_MAP makes more sense (from an English perspective, but also if we have multiple constants directly related to feature flags, its easier to read/group like `FEATURE_FLAGS_MAP`, `FEATURE_FLAGS_OPTIONS`, etc

Reply:
This means feature flags for `core/Map`. I think we might need to define the feature flags for widget as well. But combining these flags makes the code be complicated, so I separated these implementation.

Reply:
Will the flags be different? Don't we want the map and widgets to be synchronized on what features are enabled/disabled?

Reply:
No, I mean defining the flags in same place is difficult to read the code.
```
type FEATURE_FLAGS = {
  FLAG1_FOR_MAP,
  FLAG2_FOR_MAP,
  FLAG3_FOR_MAP,
  FLAG4_FOR_MAP,
  FLAG5_FOR_MAP,
  FLAG1_FOR_WIDGET,
  FLAG2_FOR_WIDGET,
  FLAG3_FOR_WIDGET,
};
```

Reply:
Fixed!: https://github.com/reearth/reearth/pull/507/commits/a969a5ffd1e6e1c6bbdd83446ba49c7d6d211a53